src/neo4j_genai/generation/prompts.py

```diff
@@ -151,5 +184,10 @@ class ERExtractionTemplate(PromptTemplate):
 """
     EXPECTED_INPUTS = ["text"]
 
-    def format(self, text: str, schema: dict[str, Any], examples: str) -> str:
+    def format(
+        self,
+        schema: dict[str, Any],
+        examples: str,
+        text: str = "",
```

Comment:
Maybe a detail, but `text` is not optional anymore, right?

Reply:
Nope it's not optional, i can remove the default if you'd like.